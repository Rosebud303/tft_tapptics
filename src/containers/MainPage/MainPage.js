import React, { Component } from 'react';
import ChosenDisplay from '../ChosenDisplay/ChosenDisplay';
import CharacterChoices from '../CharacterChoices/CharacterChoices'

export default class MainPage extends Component {
  render() {
    return (
      <div>
        <ChosenDisplay />
        <CharacterChoices />
      </div>
    )
  }
}
