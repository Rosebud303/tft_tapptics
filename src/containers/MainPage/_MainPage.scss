.MainPage {
  background-image: linear-gradient($fade1, $fade2), url('../../assets/images/summonersrift.jpg');
}