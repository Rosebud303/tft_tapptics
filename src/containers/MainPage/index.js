import React, { Component } from 'react';
import ChosenDisplay from '../ChosenDisplay';
import CharacterChoices from '../CharacterChoices'

export default class MainPage extends Component {
  render() {
    return (
      <div>
        <ChosenDisplay />
        <CharacterChoices />
      </div>
    )
  }
}
