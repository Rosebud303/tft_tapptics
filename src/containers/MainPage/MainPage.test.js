import React from 'react';
import { shallow } from 'enzyme';
import MainPage from './index';

describe('MainPage', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MainPage/>);
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});