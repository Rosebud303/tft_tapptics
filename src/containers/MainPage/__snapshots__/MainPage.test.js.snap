// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage should match the snapshot with all the data passed in 1`] = `
<div
  className="MainPage"
>
  <ChosenDisplay />
  <CharacterChoices />
</div>
`;
